### DISCUSSION TITLE
CSV columns should always be string-valued

### DISCUSSION THREAD

**Discussion by kerinin** [kaskada-ai/kaskada#229] [Ideas]
When loading two different portions of a CSV file, it's possible to create inconsistent DB schemas. I propose that we always treat CSV columns a strings.

**Comment by bjchambers** [comment]
This could make sense, but would likely have efficiency problems -- manipulating strings is far more costly than manipulating integers. An alternative would be to default to treating them as strings, but allow the user to specify a schema. And/or make it configurable with some options:

1. Use *this* schema ...
2. Default to strings
3. Default to inferred types
